updates_from_googleapis.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+OWLBOT_VERSION=latest
+
+
+# ROOT_DIR must have googleapis and google-cloud-java directories
+ROOT_DIR=$HOME
+
+set -ef
+
+cd $ROOT_DIR/googleapis
```

Comment:
TODO: read the versions from the "dependency version file" (which we need to design).